superset-frontend/src/SqlLab/components/SouthPane/SouthPane.tsx

```diff
@@ -33,7 +33,8 @@ import {
   LOCALSTORAGE_MAX_QUERY_AGE_MS,
 } from '../../constants';
 
-const TAB_HEIGHT = 64;
+const TAB_HEIGHT = 90;
+const RESULTS_TAB_HEIGHT = TAB_HEIGHT - 10;
```

Comment:
nice catch to make this a variable. But are they really related? Or is it just coincidence that they are 10 pixels apart? If they are separate, I would just say `RESULTS_TAB_HEIGH = 80;`

Reply:
@eschutho I was wondering about that. Originally they were the same height, so I thought that they might be related, so that's why I had them related, so you only need to change one variable.

Reply:
Functionally they are not related though